scripts/tools/zap/run_zaptool.sh

```diff
@@ -39,6 +42,18 @@ function _print_usage() {
         echo " General Options:"
         echo ""
         echo "  -h, --help                  Display this help, then exit."
+        echo "  --zap ZAP                   Use the zap program ZAP as he zap executable to "
```

Comment:
```suggestion
        echo "  --zap ZAP                   Use the zap program ZAP as the zap executable to "
``

Reply:
Good catch, thanks. Addressed and pushed.